### PR TITLE
Add SHR tests from eip-145

### PR DESCRIPTION
**Pull request by pirapira**
Taken from https://github.com/ethereum/EIPs/blob/master/EIPS/eip-145.md

### CODE REVIEW

**Comment by pirapira** [comment]
Shortened test names.

File: src/GeneralStateTestsFiller/stShift/shr_2^255_257Filler.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                        "storage" : {
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+            "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
+                "balance" : "1000000000000000000",
+                "code" : "PUSH32 0x8000000000000000000000000000000000000000000000000000000000000000; PUSH2 0x0101; SHR; PUSH1 0; SSTORE",
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stShift/shr_2^255_257Filler.json]: comments //

Comment (winsvega) [reply src/GeneralStateTestsFiller/stShift/shr_2^255_257Filler.json]: some json parsers eroors if file contains multimple fields with the same name

**Comment by winsvega** [comment]
we should put a link to tests google doc to this PR 
so we knew where to find implementation of the tests for this EIP

https://docs.google.com/spreadsheets/d/1xat7UI8GtB4ZGVdlK5_XQSHJZaMThi4SrlcL8XMZb5Q/edit#gid=1773257006

**Comment by pirapira** [comment]
@winsvega why don't we put the link directly in the EIP?

File: src/GeneralStateTestsFiller/stShift/shr_2^255_257Filler.json (hunk 1 above)
Comment (pirapira) [reply src/GeneralStateTestsFiller/stShift/shr_2^255_257Filler.json]: I modified these duplicate keys.

**Comment by winsvega** [comment]
because there are so many EIP and it is such a mess I can't tell which is implemented, rejected or just a proposal. and I dont even know where the latest eip thread is...

**Comment by pirapira** [comment]
@winsvega file a PR against this file https://github.com/ethereum/EIPs/blob/master/EIPS/eip-145.md#implementation

**Comment by winsvega** [comment]
what you mean? add a link to this PR in that .md file?

**Comment by pirapira** [comment]
@winsvega add a link to the directory `stShift`.

**Comment by winsvega** [comment]
https://github.com/ethereum/EIPs/pull/944

**Pull request merged**